docs/querying/basics.md

```diff
@@ -109,8 +109,16 @@ single sample value for each at a given timestamp (point in time).  In the simpl
 form, only a metric name is specified, which results in an instant vector
 containing elements for all time series that have this metric name.
 
+The value returned will be that of the most recent sample prior to the query's
```

Comment:
It's the sample exactly at the time, or prior, no?

Reply:
Right. It's not "prior". Maybe "at or before" would be the suitable wording?

Reply:
```
-The value returned will be that of the most recent sample prior to the query's
-evaluation timestamp (in the case of an
+The value returned will be that of the most recent sample at or before the
+query's evaluation timestamp (in the case of an
 [instant query](api.md#instant-queries))
-or prior to the current step within the query (in the case of a
+or the current step within the query (in the case of a
 [range query](api.md/#range-queries)).
```